src/main/java/trade/wayruha/mexc/ws/WSClientFactory.java

```diff
@@ -30,25 +36,137 @@ public WSClientFactory(ApiClient apiClient) {
         this.objectMapper = new ObjectMapper();
     }
 
-    public <T> WebSocketClient<T> orderBookSubscription(Set<String> symbols, int depth, WebSocketCallback<T> callback, Class<T> type) {
+    //<editor-fold desc="Public methods">
```

Comment:
why do we have this comments? what for?

Reply:
In order to delimit public and private(personal) method. It's convenient thing in IDE

Reply:
which IDE do you use? cause my IDEA does not recognize this comments and just displays it as plain text comment